.eslintrc.js

```diff
@@ -38,5 +39,32 @@ module.exports = {
         ],
         'no-return-assign': ['error', 'always'],
         'no-implicit-coercion': ['error', {allow: ['!!']}],
+
+        '@nrwl/nx/enforce-module-boundaries': [
+            'warn', // TODO: increase error level as soon as warnings are fixed
```

Comment:
Hello, for some reason there are no warnings in the logs, is it correct?

![image](https://user-images.githubusercontent.com/12021443/151653360-8e7c4e95-5f27-4135-92d0-aa3b05ffaefd.png)

Reply:
Definitely not. Please check that you have a clean copy of the branch, run `npm ci` and restart the Nx daemon with `nx reset`. After killing the daemon this rule might not work at all until you perform any build, e.g. with `npm start`. 
There should be over 200 warnings with this rule.

Reply:
`200 warnings` are you sure they are problems?

Reply:
I'm pretty sure that there's a lot to consider. E.g. we have a lot of imports from the `demo` in `addon-doc` tests.
Of course I didn't do manual checks for every warning but looked into a couple of them and at least 20-30% of these should be fixed.

Reply:
Thank you for explain!